main.py

```diff
@@ -742,22 +742,29 @@ def make_move(move, board:Board):
     #update king position
     if board.board[position] == K or board.board[position] == k:
         board.king_position[board.side] = target
-    else:
+    #update other pieces positions
+    elif board.board[position] != K or board.board[position] != k:
         if char_ascii[board.board[position]] == '.': 
             board.undo_move()
             return 0  #the second check. first was in line 730
+        if capture:
+            captured_piece_list_position = get_piece_positions_from_letter(char_ascii[board.board[target]],board)
+            captured_piece_list_position.remove(target)
+
```

Comment:
are you using this for the evaluation function? otherwise i don't really see a reason why we should save the captured pieces

Reply:
it does not save captured pieces. lets look at this position:
![image](https://github.com/Garic152/pythonchess/assets/85243017/27c2a683-2916-455c-83c6-e17598a34443)

first we have now positions_of_horses = [d8 (as integer),f7(as integer) ]  and positions_of_queen = [f3 (as integer)]
what it does is  in line750 when we have a capture (0 or 1), for example queen captures the knight and 
it will take the positions of knights [d8, f7]  (in line 751)
and remove f7 from there (in line 752) => now in positions_of_knights we have only [d8] with these 3 lines of code

Reply:
Okay wasn't sure what get_piece_positions_from_letter does but yeah that makes sense 👍